pkg/conn/conn_test.go

```diff
@@ -136,3 +136,13 @@ func (s *testClientSuite) TestGetAllTiKVStores(c *C) {
 		c.Assert(foundStores, DeepEquals, testCase.expectedStores)
 	}
 }
+
+func (s *testClientSuite) TestGetConnOnCancelledContext(c *C) {
```

Comment:
```suggestion
func (s *testClientSuite) TestGetConnOnCanceledContext(c *C) {
```

Cancelled is British English, but [TiKV style guide](https://github.com/pingcap/style-guide/blob/master/docs/rust/comments.md#language-for-a-good-comment) told us to use American English...